x-pack/test/plugin_api_perf/test_suites/task_manager/task_manager_perf_integration.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from '@kbn/expect';
+import url from 'url';
+import supertestAsPromised from 'supertest-as-promised';
+
+export default function({ getService }: { getService: (service: string) => any }) {
+  const log = getService('log');
+  const config = getService('config');
+  const supertest = supertestAsPromised(url.format(config.get('servers.kibana')));
+
+  describe('stressing task manager', () => {
+    beforeEach(() =>
+      supertest
+        .delete('/api/sample_tasks')
```

Comment:
I don't see an implementation of this route.  Is it implemented somewhere else?

Reply:
oops, left overs from splitting  this out of the other test file.

Cleaned up now,